lib/gui/etcher.js

```diff
@@ -110,7 +125,7 @@ const createMainWindow = () => {
     autoUpdater.on('error', (err) => {
       analytics.logException(err)
     })
-    if (settings.get('updatesEnabled')) {
+    if (packageUpdatable && settings.get('updatesEnabled')) {
```

Comment:
We should remove the test for `settings.get('updatesEnabled')` because it might be changed later and is already checked in `checkForUpdates`